Jenkinsfiles/utils.groovy

```diff
@@ -7,6 +7,18 @@ STAGE_BRANCH_NAME = 'stage-push'
 STANDBY_BRANCH_NAME = 'standby-push'
 KUMA_PIPELINE = 'mdn_multibranch_pipeline'
 KUMASCRIPT_PIPELINE= 'kumascript_multibranch_pipeline'
+FUTURE_KUMA_PIPELINE = 'kuma'
+FUTURE_KUMASCRIPT_PIPELINE= 'kumascript'
```

Comment:
Nits: When I'm migrating from one thing to another, I avoid words like "new" and "future" that look silly when they hang around for months and years. 

I make the future thing the standard, and the past the qualified one. So I would use:

```
MOZMEAO_KUMA_PIPELINE = 'mdn_multibranch_pipeline'
MOZMEAO_KUMASCRIPT_PIPELINE = 'kumascript_multibranch_pipeline'
KUMA_PIPELINE =  'kuma'
KUMASCRIPT_PIPELINE = 'kumascript'
```

Then, when we drop support for the MozMEAO infrastructure, we would just delete code paths rather than delete and modify.